source/geometry_model/chunk.cc

```diff
@@ -270,10 +270,9 @@ namespace aspect
 #endif
 
       // Transform box into spherical chunk
-      GridTools::transform (std::bind(&ChunkGeometry::push_forward,
-                                      std::cref(manifold),
-                                      std::placeholders::_1),
-                            coarse_grid);
+      GridTools::transform ([&](const Point<dim> &p) -> Point<dim>
```

Comment:
Here the lambda return type and arguments are in the line with the opening parentheses, above they are on a separate line. I think we should make this as explicit as possible, maybe it is just me, but I find lambda syntax not very intuitive. What about the following convention (you use this in sevaral places already):
```
GridTools::transform (
  [&](const Point<dim> &p) -> Point<dim>
  {
    return manifold.push_forward(p);
  },
  coarse_grid);
```
This seems most easy to understand to me.